docs/jobs/job-writing-guide.md

```diff
@@ -1131,7 +1128,7 @@ fn(state => {
 });
 ```
 
-You could use the spread operator to override some keys:
+You could use the _spread_ operator to override some keys:
```

Comment:
Ah I'm not sure about italicising the operator names like this. I don't think we do it anywhere else in the doc  site, and it's not a convention I recognise.

If anything I'd use propercase, ie, Spread, Rest. 

But [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) doesn't add any emphasis or capitalisation, and I'm happy just to follow their lead on this.

Reply:
IMO and following general writing rules, adding emphasis makes it way easier to read and know it is an operator, especially for those who are less code familiar. However, again I, happy to change to get this merged.